<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equipe_window</class>
 <widget class="QMainWindow" name="equipe_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1554</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>equipe_window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton {
    color: white;
}
QRadioButton::indicator:unchecked {
    border: 2px solid gray;
    background: darkgray;
}
QRadioButton::indicator:checked {
    border: 2px solid red;
    background: blue;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1521</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Liste_equipe">
     <attribute name="title">
      <string>Liste des equipes</string>
     </attribute>
     <widget class="QTableView" name="Liste_equipes">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>871</width>
        <height>631</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>40</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ajouter_eq">
     <attribute name="title">
      <string>Ajouter une equipe</string>
     </attribute>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_equipe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Participations</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Moyen de buts</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Buteur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Total de matches</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom equipe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_equ">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nom_eq">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Participation_ajout">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="moy_buts_ajout">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="buteur_ajout">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tot_matchs_ajout">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>400</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="valider_ajout">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>370</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider Ajout</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Classe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Classe_ajout">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>460</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Modifier_equipe">
     <attribute name="title">
      <string>Modifier equipes</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Participations</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Buteur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Total des matches</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Moyen de buts</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="participations_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Buteur_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Total_matches_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Moyen_but_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>350</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="valider_modifier">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>360</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider Modification</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Classe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Classe_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>440</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Supprimer_equipe">
     <attribute name="title">
      <string>Supprimer une equipe</string>
     </attribute>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Voulez vous supprimer cet equipe?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Valider_supprimer">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider suppression</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tirage au sort</string>
     </attribute>
     <widget class="QPushButton" name="Tirageausort">
      <property name="geometry">
       <rect>
        <x>1270</x>
        <y>530</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Tirage au sort</string>
      </property>
     </widget>
     <widget class="QTableView" name="groupe2">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>160</y>
        <width>321</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="groupe3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>160</y>
        <width>321</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="groupe1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>321</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="groupe4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>341</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Groupe1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Groupe2:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>70</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Groupe3:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>60</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Groupe4:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Valeur marchande d'une equipe</string>
     </attribute>
     <widget class="QLineEdit" name="valeurmarchande">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Voici la valeur marchande de l'equipe</string>
      </property>
     </widget>
     <widget class="QTableView" name="joueursList">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>591</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="rechercher_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider recherche</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche d'equipe par ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="recherche">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1554</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
